tests/integration_tests/data_steward/validation/ehr_union_test.py

```diff
@@ -20,6 +20,7 @@
 
 PITT_HPO_ID = 'pitt'
 NYC_HPO_ID = 'nyc'
+THIRD_HPO_ID = 'fake'
```

Comment:
Can you name this `EXCLUDED_HPO_ID`?
```suggestion
EXCLUDED_HPO_ID = 'excluded'
```